lib/netext/tracer.go

```diff
@@ -107,6 +114,9 @@ func (t *Tracer) Done() Trail {
 	if !t.connectDone.IsZero() && !t.connectStart.IsZero() {
 		trail.Connecting = t.connectDone.Sub(t.connectStart)
 	}
+	if !t.tlsHandshakeDone.IsZero() && !t.tlsHandshakeStart.IsZero() {
+		trail.TLSHandshaking = t.tlsHandshakeDone.Sub(t.tlsHandshakeStart)
+	}
 	if !t.wroteRequest.IsZero() {
 		trail.Sending = t.wroteRequest.Sub(t.connectDone)
```

Comment:
If `t.tlsHandshakeDone` is not zero (the request was sent over TLS) we should sub `t.tlsHandshakeDone` here rather than `t.connectDone`, so need an if/else. This seems consistent with how Chrome reports "SSL" and "Request sent" in the devtools for example.

Reply:
Just so we are clear, you are saying to calculate `trail.Connecting` by subtracting `t.connectStart` from `t.tlsHandshakeDone` in the case where the request was sent over TLS?

BTW. its like that in Chrome but Firefox shows them separate (like it is right now) :)

Reply:
Ah no, I'm talking about the `trail.Sending` :) yes, connecting and TLS handshaking should be separate and afaikt from the Go net/http code the TLS handshake done-start doesn't include the TCP connection time.

Reply:
Well that makes much more sense! I'll update that ASAP.